avm/res/network/network-security-perimeter/profile/main.bicep

```diff
@@ -0,0 +1,79 @@
+metadata name = 'Network Security Perimeter Profile'
+metadata description = 'This module deploys a Network Security Perimeter Profile.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Conditional. The name of the parent network perimeter. Required if the template is used in a standalone deployment.')
+param networkPerimeterName string
+
+@maxLength(64)
+@description('Required. The name of the network security perimeter profile.')
+param name string
+
+@description('Optional. Static Members to create for the network group. Contains virtual networks to add to the network group.')
+param accessRules accessRulesType?
+
+resource networkSecurityPerimeter 'Microsoft.Network/networkSecurityPerimeters@2023-08-01-preview' existing = {
+  name: networkPerimeterName
+}
+
+resource networkSecurityPerimeter_profile 'Microsoft.Network/networkSecurityPerimeters/profiles@2023-08-01-preview' = {
+  name: name
+  parent: networkSecurityPerimeter
+}
+
+resource nsp_accessRules 'Microsoft.Network/networkSecurityPerimeters/profiles/accessRules@2023-08-01-preview' = [
```

Comment:
Would be great if this could be its own child-module - soon enabling you to publish it as its own module.